pkgs/development/tools/misc/pinact/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, fetchFromGitHub
+, buildGo120Module
+, testers
+, pinact
+}:
+
+buildGo120Module rec {
```

Comment:
Agreed. Replaced to `let ... in` as https://github.com/NixOS/nixpkgs/pull/261397/commits/b819b61d91b61789e3f59005ebb0645c07cf7e73#diff-fa2d5b5d9f78fd26b25385f9f0f04acd1f4b57d2f8cac71b96faa217dc47c2c1R8-R19

Reply:
> suggestion: instead of `rec`, use `let ... in` bindings.

Why?

Reply:
> > suggestion: instead of `rec`, use `let ... in` bindings.
> 
> Why?

This was a [suggestion](https://conventionalcomments.org/), hence the prefix.

- I find `let ... in` or `finalAttrs:` clearer/easier to read than recursive attrsets.
- `rec` can sometimes lead to problems. Not in this case, but avoiding its use in general makes it less likely that Nix newcomers will copy it.